.github/workflows/main.yml

```diff
@@ -34,12 +34,21 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v1
 
+    - name: Edit Snapfile
+      run: echo "devices([\"iPhone 8\"]);languages([\"en-US\"])" > fastlane/Snapfile
```

Comment:
Is this really needed? I would love to use it without this, because now, we have to define the language and test device on two different places (`Fastfile` and here)...

Reply:
The problem is that fastlane fails to run a snap command if the snap file contains a device that is not available on the CI (I this case the Se) even if the the devices isn‘t required in the snap command. Not sure if we can delete the snap file completely.

I’ll try that if that doesn’t work I’ll add a comment to explain why this line is needed

Reply:
What do you think about moving all the code from the Snapfile to the Fastfile? So we have all the devices and languages defined in the Fastfile.

Reply:
looks way better now!